tests/integration/golang/aim/run/search_test.go

```diff
@@ -25,7 +31,303 @@ func TestSearchTestSuite(t *testing.T) {
 	suite.Run(t, new(SearchTestSuite))
 }
 
-func (s *SearchTestSuite) Test_Ok() {
+func (s *SearchTestSuite) TestCSVReport_Ok() {
+	// create test experiments.
+	experiment, err := s.ExperimentFixtures.CreateExperiment(context.Background(), &models.Experiment{
```

Comment:
could all this setup be shared with `TestStreamData_Ok` by using setup method?

Reply:
ok, should be fine now.